pootle/apps/accounts/adapter.py

```diff
@@ -47,3 +48,12 @@ def is_open_for_signup(self, request):
     def add_message(self, request, level, message_template, *args, **kwargs):
         """Silence messages altogether."""
         pass
+
+    def send_confirmation_mail(self, *args, **kwargs):
+        try:
+            super(PootleAccountAdapter, self).send_confirmation_mail(*args,
+                                                                     **kwargs)
+        except Exception:
+            raise RuntimeError(_('Some problem happened when tried to send '
+                                 'the confirmation email. Please try again '
+                                 'later.'))
```

Comment:
I think we should comunicate that their signup was successful - but just not sending conf email

Otherwise they might try to sign up again - but they already have an account